pdns/recursordist/rec-main.cc

```diff
@@ -2084,8 +2084,9 @@ static void houseKeeping(void*)
     // Below are the thread specific tasks for the handler and the taskThread
     // Likley a few handler tasks could be moved to the taskThread
     if (info.isTaskThread()) {
+      cerr << time(nullptr) << " T" << endl;
```

Comment:
I'm pretty sure you did not mean to commit this :)

Reply:
Oops, it really is Friday afternoon